notify/discord.py

```diff
@@ -0,0 +1,78 @@
+import json
+import logging
+import os
+import requests
+
+states = json.loads(os.environ["STATES"])
+
+
+def format_available_message(locations):
+    message = ":green_circle: Vaccine appointments available at {} location{}:".format(
```

Comment:
I don't use discord, but can you verify if this puts the emoji in the message correctly like slack does?

Reply:
That it does. Below is example of posting.

![image](https://user-images.githubusercontent.com/6699852/110732950-ba213180-81ea-11eb-8db1-71e8863e9323.png)

Reply:
please note on the url %7Chere is fixed. This screenshot was before that change.

Reply:
`*test*` does bold in slack, it does italic in [Discord markdown](https://support.discord.com/hc/en-us/articles/210298617-Markdown-Text-101-Chat-Formatting-Bold-Italic-Underline-)
`**test**` does bold

Reply:
If possible, you could do a link like how slack has it:
https://stackoverflow.com/questions/63160401/how-to-create-hyperlink-in-discord-in-an-embed-in-general
![image](https://user-images.githubusercontent.com/5343931/110733465-a1fde200-81eb-11eb-8c6c-036b91a4d5c7.png)

If not possible, you're welcome to use the tinyurl shortener that twitter.py uses

Reply:
I did check, that markdown feature is not available in discord. I will put in a shortened url.

Reply:
updated...

![image](https://user-images.githubusercontent.com/6699852/110734909-5dc01100-81ee-11eb-8483-16b87669921b.png)

Reply:
changes implemented, please check formatting to make sure it looks good.